src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -1,9 +1,307 @@
 import React from 'react';
+import Cropper from 'react-cropper';
+import Dropzone from 'react-dropzone';
+import ErrorableFileInput from '../../common/components/form-elements/ErrorableFileInput';
+
+const PhotoDescription = (
+  <div className="feature">
+    <h3>What makes a good photo?</h3>
+    <div>
+      <p>To meet the requirements for a Veteran ID Card, your photo should follow the guidance listed below. It must:</p>
+      <ul>
+        <li>Show a full front view of your face and neck, (with no hat, head covering, or headphones covering or casting shadows on your hairline or face), <strong>and</strong></li>
+        <li>Be cropped from your shoulders up (much like a passport photo), <strong>and</strong></li>
+        <li>Show you with your eyes open and a neutral expression, <strong>and</strong></li>
+        <li>Be a square size and have a white or plain-color background (with no scenery or other people in the photo)</li>
+        <li>Be uploaded as a .jpeg, .png, .bmp, or .tiff file</li>
+      </ul>
+      <h3>Examples of good ID photos</h3>
+      <img className="example-photo" alt="placeholder" src="/img/example-photo-1.png"/>
+      <img className="example-photo" alt="placeholder" src="/img/example-photo-2.png"/>
+    </div>
+  </div>);
+
+const FILE_TYPES = [
+  'png',
+  'tiff',
+  'tif',
+  'jpeg',
+  'jpg',
+  'bmp'
+];
+const MIN_SIZE = 350;
+
+function detectDragAndDrop() {
+  const div = document.createElement('div');
+  const supportsDragAndDrop = ('draggable' in div) || ('ondragstart' in div && 'ondrop' in div);
+  const iOS = !!navigator.userAgent.match('iPhone OS') || !!navigator.userAgent.match('iPad');
```

Comment:
Probably not widely used but iPads support drag and drop on iOS 11.

Reply:
Thanks @rianfowler, I'll include looking into this in the ticket for work after user testing.